scripts/kc_data_transfer/readme.md

```diff
@@ -28,29 +28,38 @@ forest client roles. To change this modify the method `KeyCloakToFAM.get_fam_for
 
 ```
 cd scripts/kc_data_transfer/
-python3.8 -m venv venv
+python3 -m venv venv
 . ./venv/bin/activate
 pip install -r src/requirements.txt
 ```
 
 ## Populate the following environment variables
 
-Either set these env vars or stuff them into .env file.  If the .env file
```

Comment:
Maybe mention that there is an example file that can be renamed. Also a STRONG warning not to check in the secret